test/blackbox-tests/test-cases/env-cflags/run.t

```diff
@@ -0,0 +1,47 @@
+  $ dune printenv --profile default .
+  (
+   (flags (-w -40))
+   (ocamlc_flags (-g))
+   (ocamlopt_flags (-g))
+   (c_flags (":standard + in ."))
+   (cxx_flags (":standard + in ."))
+  )
+
+  $ dune printenv --profile default src
+  (
+   (flags (-w -40))
+   (ocamlc_flags (-g))
+   (ocamlopt_flags (-g))
+   (c_flags (":standard + in ." ":standard + in src"))
+   (cxx_flags (":standard + in ." ":standard + in src"))
+  )
+
+  $ dune printenv --profile default bin
+  (
+   (flags (-w -40))
+   (ocamlc_flags (-g))
+   (ocamlopt_flags (-g))
+   (c_flags ("in bin"))
+   (cxx_flags ("in bin"))
+  )
+
+  $ dune printenv --profile default run
+  (
+   (flags (-w -40))
+   (ocamlc_flags (-g))
+   (ocamlopt_flags (-g))
+   (c_flags (-DTEST_C))
+   (cxx_flags (-DTEST_CPP))
+  )
+  $ dune build --profile default run/foo.exe
```

Comment:
It's a bit simpler to just do `$ dune exec --profile default ./run/foo.exe`. This will build & run the binary and you won't need the dedicated alias anymore.